### ISSUE TITLE
Splitting projects in the `package` directory

### ISSUE THREAD

**Issue by kecrily** [gridsome/gridsome#1549]
## Summary

There are now twenty subfolders under the package folder, which is a huge burden for maintainability. It's time to split them up.

### Motivation

Create new repositories and take the packages where they belong. Get gridsome/gridsome back to what it is.

**Comment by hjvedvik** [comment]
@kecrily I totally agree :) I also think it would be easier for others to take responsibility for a single plugin if it had its own repository with its own issues etc. Maybe we should create a repository template that would be a good starting point.
